doc/api_assets/style.css

```diff
@@ -488,3 +488,21 @@ th > *:last-child, td > *:last-child {
     font-size: 3.5em;
   }
 }
+
+
+@media print {
+
+  html  {
+    height : auto;
```

Comment:
Nit: the rest of the file does not have spaces before colons (`:`), so this probably also shouldn't.